CHANGELOG.md

```diff
@@ -18,6 +18,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
      - Enclave apps that are built with Make and rely on Open Enclave's pkgconfig must now explicitly include OE crypto wrapper library in linker dependency flags.
      - See the [Makefile in the helloworld sample](samples/helloworld/enclave/Makefile#L34) for an example. Here `OE_CRYPTO_LIB` is set to `mbedtls` in [parent MakeList file](samples/helloworld/Makefile#L9).
 
+### Changed
+- Syscalls are internally dispatched directly to their implementation functions instead of via a switch-case.
```

Comment:
```suggestion
- Syscalls are internally dispatched directly to their function implementations instead of via a switch-case.
```

Reply:
`implementation function` is a more common usage than `function implementation`. Retaining existing wording.